### ISSUE TITLE
pipx list command tries to upgrade packages

### ISSUE THREAD

**Issue by zeitgenosse** [pypa/pipx#1564]
**Describe the bug**

According to man pages and `--help`, `pipx list` is supposed to show a list of installed packages. Instead, it attempts to upgrade shared libraries, which can take a long time when on an very slow internet connection (and unnecessarily use data on a metered connection).

**Expected behavior**

`pipx list` should immediately list all locally installed packages and not attempt to upgrade libraries.

**Comment by dukecat0** [comment]
Which version of pipx are you using?

**Comment by zeitgenosse** [comment]
Version is 1.1.0, an ancient version shipped with Debian stable and derivatives.

I did search through open issues, but I did the mistake of only looking at bugs and not feature requests as well, so I overlooked issue #1081. I've found [pull request 1163](https://github.com/pypa/pipx/pull/1163) introducing `--skip-maintenance`, so this bug is obsolete. Sorry for wasting your time.

I do however think that `--skip-maintenance` should be the default behavior, but a simple alias in my .bashrc is good enough for me.
